packages/playground/src/utils/validators.ts

```diff
@@ -6,7 +6,7 @@ import type { IsURLOptions } from "validator/lib/isURL";
 
 export function required(msg: string) {
   return (value: string) => {
-    if (value === "" || value === undefined || value === null) {
+    if (value === undefined || value === null) {
```

Comment:
why u changed this function ?  It affects all fields also it's used a lot in the playground.

- i think u should remove required from the HRU and inlcude `validators.min` in rules.
- also if the user removed the value, consider it as zero.